### PR TITLE
Cloudlab working

### PR DESCRIPTION
**Pull request by edunextgensairam**
cloudlab is working but still changes requried And ui want to change

### CODE REVIEW

File: src/app/portal/cloudeditor/cloudlab/cloudlab.component.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+div.code-editor { max-height: 70vh; }
+
+div.label-box {
+    border-top-left-radius: .5rem!important;
```
Comment (vivektiwariedunextgen) [inline src/app/portal/cloudeditor/cloudlab/cloudlab.component.scss]: avoid using `important` use specificity

**Comment by edunextgensairam** [comment]
cloudlab working with changed ui

**Comment by vivektiwariedunextgen** [comment]
please resolve the conflict and erase the commented code

**Comment by edunextgensairam** [comment]
cloudlab merged to master

**Comment by edunextgensairam** [comment]
cloud lab is merged to master

**Pull request merged**